addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/config/NetatmoWelcomeConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.config;
+
+/**
+ * The {@link NetatmoWelcomeConfiguration} is responsible for holding configuration
+ * informations needed to access a Netatmo Welcome Camera
+ *
+ * @author Ing. Peter Weiss - Initial contribution
+ */
+public class NetatmoWelcomeConfiguration {
```

Comment:
the only Id which is an equipment ID is the cameraID, all other Ids are not really equipments so this wouldn't match, ill check to implement a Basic Super Class for all things and then split into the different devices with Abstract classes for them